apps/01PigLatin.js

```diff
@@ -9,9 +9,40 @@ var rl = readline.createInterface({
 
 
 function pigLatin(word) {
+word = word.toLowerCase();
+var firstVowel = -1;
+
+if (firstVowel === 0) {
```

Comment:
debug this. you are assigning -1 to the variable but then immediately checking if it is 0 on the next line. Will it ever be 0 ?